packages/x-components/src/x-modules/related-tags/wiring.ts

```diff
@@ -95,6 +95,9 @@ export const setUrlParamsWire = wireDispatch('setUrlParams');
  * @internal
  */
 export const relatedTagsWiring = createWiring({
+  ParamsLoadedFromUrl: {
```

Comment:
Was this change on purpose? I think it might be weird to check the blame of these lines later in time and see the commit they were changed in 😅

Reply:
yes, it is. Actually we found a bug out of this change https://searchbroker.atlassian.net/browse/EX-5327